src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -857,7 +857,8 @@ get_tok(char **src, char *buf, int buflen, char sep)
 static void
 printUsage(void)
 {
-     TTY_MESSAGE((
+    // SapMachine 2022-12-12 Revert JDK-8226608, we should show the onjcmd=y|n option in jdwp usage
+    TTY_MESSAGE((
```

Comment:
Indentation is changed. Would the comment be better directly before the new line?

Reply:
No, comments between the string parts are ok.

Reply:
OK, let's try it then. I force-pushed the update.